tools/flake8.ini

```diff
@@ -10,14 +10,15 @@ select = E,W,F,N
 # E303: too many blank lines (3)
 # E305: expected 2 blank lines after end of function or class
 # E402: module level import not at top of file
+# E501: line too long
```

Comment:
I don't think we want to ignore this for non-WebDriver files? I think you should be able to use [`per-file-ignores`](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-per-file-ignores) to just apply this to files under `webdriver` for now?

Reply:
I would also like to leave that in for webdriver tests. Is there a reason why those long lines should be ignored?

Reply:
The reason for ignoring is that there are too many violations of "line too long" in `webdriver/`. I did not want to make the diff even noisier than it already is.

I suppose `per-file-ignores` is a good compromise here. We could trivially fix all "line too long" occurrences later on  once we agree upon a formatter.

Reply:
Would for an initial run maybe black help here? We could run it once before this PR gets merged. Usually we obeyed the 100 chars line length, so it's interesting to hear that many files are failing.

Reply:
Done.

There were many occurrences, but they were mostly in a small number of files. I just formatted those files. It seems all E501 violations are now fixed.